parser/scanner.l

```diff
@@ -492,9 +493,26 @@ sys_var		"@@"(({global}".")|({session}".")|{local}".")?{ident}
 
 \"			l.sc = S1
 '			l.sc = S2
-
-<S1>(\\.|[^\"]|\"\")*\"		return l.str(lval, "\"")
-<S2>((\\')|[^']|\n|'')*'	return l.str(lval, "'")
+`			l.sc = S4
+
+<S1>[^\"\\]*		l.stringLit = append(l.stringLit, l.val...)	
+<S1>\\.			l.stringLit = append(l.stringLit, l.val...)
```

Comment:
above line handles not " and \\, this line handles \\ followed by any character, they are not same, and I think distinguish them is better to know.